services/src/main/scala/cromwell/services/metadata/impl/WriteMetadataActor.scala

```diff
@@ -38,12 +38,36 @@ class WriteMetadataActor(override val batchSize: Int,
       case Success(_) =>
         putWithResponse foreach { case (ev, replyTo) => replyTo ! MetadataWriteSuccess(ev) }
       case Failure(regerts) =>
-        putWithResponse foreach { case (ev, replyTo) => replyTo ! MetadataWriteFailure(regerts, ev) }
+
+        val (outOfLives, stillGood) = e.toVector.partition(_.ttl <= 1)
+
+        handleOutOfLives(outOfLives, regerts)
+        handleEventsToReconsider(stillGood)
     }
 
     dbAction.map(_ => allPutEvents.size)
   }
 
+  private def handleOutOfLives(writeActions: Vector[MetadataWriteAction], reason: Throwable): Unit = {
+    val workflowMetadataFailureCounts = writeActions.toVector.flatMap(_.events).groupBy(x => x.key.workflowId).map { case (wfid, list) => s"$wfid: ${list.size}" }
+    log.error("Metadata events have been permanently dropped for the following workflows: " + workflowMetadataFailureCounts.mkString(","))
```

Comment:
Suggesting a word more alarming than "dropped", because dropped is also used in contexts where it's deliberate - e.g. DB tables
```suggestion
    log.error("Metadata events have been irretrievably lost for the following workflows: " + workflowMetadataFailureCounts.mkString(","))
```